api/EventTarget.json

```diff
@@ -739,6 +739,40 @@
             }
           }
         }
+      },
+      "when": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/observable/#dom-eventtarget-when",
+          "support": {
+            "chrome": {
+              "version_added": "135"
+            },
+            "chrome_android": "mirror",
```

Comment:
According to https://chromestatus.com/feature/5154593776599040, it's only shipping on desktop, not Android/WebView.

Reply:
Oh that's surprising. @domfarolino, is the Observable API shipping on desktop only?

Reply:
Thanks for pinging me. It is shipping on all platforms that Blink is supported on, including Android, so this is wrong. Was this populated automatically from the Chrome status? I've amended https://chromestatus.com/feature/5154593776599040 to reflect that it is shipping on Android, hopefully that helps.

Reply:
It was automatically collected using https://github.com/openwebdocs/mdn-bcd-collector. We look at Chrome Status for double-checking only. 

You say "shipping on all platforms that Blink is supported on", does that include Android WebView? You've left that blank on Chrome Status.

Reply:
I've fixed that too now. I was told that checking the "all platforms" box would cover that, but apparently not, so I've done it manually. Thanks for the heads up.

Reply:
Thanks @domfarolino!

@caugner I think this PR is correct as is then.